dashboard/app/assets/stylesheets/application.scss

```diff
@@ -3967,4 +3967,39 @@ html[dir=rtl] {
   }
 }
 
+a {
```

Comment:
@levadadenys I haven't done much CSS in the context of our new design system, so LMK what you think of this -- my thought was to add a new class for a tags that look like buttons with the updated design system. Might be a preferred way to do this, let me know if so!